content/docs/welcome-guide/setup/installing-linux.md

```diff
@@ -56,3 +65,25 @@ Example of launching Project Manager from the shell:
 ```shell
 /opt/O3DE/2111_1/bin/Linux/profile/Default/o3de
 ```
+
+## Removing O3DE
+
+1. Run apt to remove o3de:
+    ```shell
+    sudo apt remove o3de
+    ```
+    An output like the following will be displayed:
```

Comment:
Seems kinda odd that after issuing a remove command, it says "Package o3de is not installed, so not removed", especially when the output from the install said "The following NEW packages will be installed: o3de".

I'm no Linux expert, but is this correct and expected? If so, perhaps it's worth pointing this out and reassuring the casual Linux user?

Reply:
oh, great catch, this was the output after removing it twice, updating